.github/workflows/closes-issue-check.yml

```diff
@@ -10,7 +10,7 @@ env:
   GH_TOKEN: ${{ github.token }}
 
 jobs:
-  closes-issue-check:
+  closes-issue-check: 
```

Comment:
nit: remove this space so this file is not changed.

Reply:
Done deal